src/Microsoft.ML.LightGBM/LightGbmCatalog.cs

```diff
@@ -98,13 +109,24 @@ public static class LightGbmExtensions
             int? numLeaves = null,
             int? minDataPerLeaf = null,
             double? learningRate = null,
-            int numBoostRound = LightGbmArguments.Defaults.NumBoostRound,
-            Action<LightGbmArguments> advancedSettings = null)
+            int numBoostRound = Options.Defaults.NumBoostRound)
         {
             Contracts.CheckValue(catalog, nameof(catalog));
             var env = CatalogUtils.GetEnvironment(catalog);
-            return new LightGbmRankingTrainer(env, labelColumn, featureColumn, groupIdColumn, weights, numLeaves, minDataPerLeaf, learningRate, numBoostRound, advancedSettings);
+            return new LightGbmRankingTrainer(env, labelColumn, featureColumn, groupIdColumn, weights, numLeaves, minDataPerLeaf, learningRate, numBoostRound);
```

Comment:
There are some pretty long lines in this file. Over the permitted length?

Reply:
i do not see any warnings related to long lines. 

i am actually reducing the length, by dropping `advancedSettings` from the line :)

---
In reply to: [251535698](https://github.com/dotnet/machinelearning/pull/2261#discussion_r251535698) [](ancestors = 251535698)